test/e2e/scenarios/ACL/acl_create_update.yaml

```diff
@@ -58,12 +58,12 @@ steps:
         userNames:
           - userone$RANDOM_SUFFIX
           - usertwo$RANDOM_SUFFIX
-    wait:
```

Comment:
```
wait:
      status:
        conditions:
          ACK.ResourceSynced:
            status: "True"
            timeout: 180
```

Why would hard coding the value give advantage over status change wait? I prefer the previous approach as it helps us catch any issue with status change flipping back and forth.

Reply:
Previous approach causes random assertion error. My assumption is there is something wrong with previous approach, and I modified to the current approach to check whether it works. And I think we need to investigate the root cause.